maven-core/src/main/java/org/apache/maven/execution/DefaultMavenExecutionRequest.java

```diff
@@ -118,6 +118,8 @@
 
     private String resumeFrom;
```

Comment:
I don't think we should split the resume-from into 2 fields.
Just add a clear javadoc saying: `null` is unused, empty unspecified.

Reply:
Alright! I've just changed this behavior.